### PR TITLE
Guild storage log Packet

### PR DESCRIPTION
**Pull request by sctnightcore**
rA PR : https://github.com/rathena/rathena/pull/3365

ty to @Lemongrass3110 / rA dev 

```
------------------------[ Guild Storage LOG ]-------------------------
#  Name                  Item-Name      Amount   Action            Time
  0 sctnightcore001    Fly Wing           50       Put          2018-07-31 01:39:33
  1 sctnightcore001    Blue Potion        10       Put          2018-07-31 01:39:20
  2 sctnightcore001    Red Potion         5        Get          2018-07-31 01:38:51
  3 sctnightcore001    Orange Potion      99       Put          2018-07-30 18:10:53
  4 sctnightcore       Red Potion         99       Put          2018-07-30 18:07:18
----------------------------------------------------------------------
```

### CODE REVIEW

File: src/Network/Receive/kRO/Sakexe_0.pm [hunk 1]
```diff
@@ -3859,4 +3860,52 @@ sub party_dead {
...
+	my %gstorage_action = (
+		0 => T('Put'),
+		1 => T('Get'),
+	);
+		
+	if ($args->{result} == 0 || $args->{result} == 1) {
+		my $message = center(T("[ Guild Storage LOG ]"), 40, '-') ."\n".
+			T("# Name				Item		Amount		Action			Time	\n");
+		for (my $i = $headerlen; $i < $msg_size; $i+=$guildstotage_len) {
+		($gstorage->{ID}, #ACCID
```
Comment (Lemongrass3110) [inline src/Network/Receive/kRO/Sakexe_0.pm]: Its not account id. Official Aegis name is "Idx"

Comment (sctnightcore) [reply src/Network/Receive/kRO/Sakexe_0.pm]: ty sir .

**Comment by moneyisgod** [comment]
How about get and put ? which situations u can use ?

get means u can get it ? and put ?  i don't got .

I like to think such as WoW Guild Bank.

**Comment by sctnightcore** [comment]
https://user-images.githubusercontent.com/3517879/43371283-d17d06bc-938e-11e8-8e0f-ee11826ae632.png

@moneyisgod  input / ouput item .

**Pull request closed without merging**